vmm/src/device_manager.rs

```diff
@@ -4977,6 +4977,10 @@ impl BusDevice for DeviceManager {
 
 impl Drop for DeviceManager {
     fn drop(&mut self) {
+        // Wake up the DeviceManager threads (mainly virtio device workers),
+        // to avoid deadlock on waiting for paused/parked worker threads.
+        self.resume().ok();
```

Comment:
lgtm - but would it not be better to log the error here?

Reply:
Right. Good catch.